database/factories/CourseFactory.php

```diff
@@ -19,7 +19,7 @@ public function definition(): array
         return [
             'name' => $this->faker->word(),
             'abbreviation' => $this->faker->lexify('????'),
-            'color' => $this->faker->word(),
+            'classes' => "bg-blue-700 text-white dark:bg-blue-800",
```

Comment:
It would be better to use a random element from an array here. Faker also has a direct method for this, which should look something like this (untested):

```PHP
'classes' => $this->faker->randomElement([
    'bg-yellow-500 text-white dark:bg-yellow-600',
    'bg-blue-700 text-white dark:bg-blue-800'
    // ... can be copied from the CourseSeeder
]),
```

Reply:
done